apps/src/music/views/MusicView.jsx

```diff
@@ -326,17 +329,19 @@ class UnconnectedMusicView extends React.Component {
     this.props.setPackId(packId);
     this.setExemplarPlaybackEvents();
 
-    this.props.isPlayView
-      ? this.musicBlocklyWorkspace.initHeadless()
-      : this.musicBlocklyWorkspace.init(
-          document.getElementById(BLOCKLY_DIV_ID),
-          this.onBlockSpaceChange,
-          this.props.isReadOnlyWorkspace,
-          toolboxAllowList,
-          this.props.isRtl,
-          this.props.blockMode,
-          localizedToolboxDefinition
-        );
+    if (AppConfig.getValue('js-editor') !== 'true') {
+      this.props.isPlayView
+        ? this.musicBlocklyWorkspace.initHeadless()
+        : this.musicBlocklyWorkspace.init(
```

Comment:
Filing for later: I've been thinking about how it would be nice to decouple the code execution elements from the Blockly-specific compilation/workspace management in `MusicBlocklyWorkspace` and seeing these conditional statements definitely enforces that. I could imagine us even abstracting this out to some common "editor" interface that is given code and outputs compiled JS (in the case of the JS editor, maybe just returning the code directly, or with some API wrappers).